http.go

```diff
@@ -92,6 +92,21 @@ func scaleForRequest(r *http.Request) (float64, error) {
 	return scale, nil
 }
 
+func pageForRequest(r *http.Request) (int64, error) {
+	// Let's first parse out some URL args and return errors if
+	// we got some bogus stuff.
+	if r.FormValue("page") == "" {
+		return -1, fmt.Errorf("Invalid page!")
+	}
+	page, err := strconv.ParseInt(r.FormValue("page"), 10, 32)
```

Comment:
`strconv.ParseInt("", 10, 32)` returns an error. Seems like the check above is redundant: https://play.golang.org/p/OPZWvlMuwM